src/window/city.c

```diff
@@ -512,6 +513,14 @@ static void handle_hotkeys(const hotkeys *h)
             }
         }
     }
+    if (h->storage_order) {
+        int grid_offset = widget_city_current_grid_offset();
+        int building_id = map_building_at(grid_offset);       
+        if (building_id) {         
+           window_building_info_show(grid_offset);
+           window_building_info_show_storage_orders();
```

Comment:
Humm. What happens if the building in question doesn't have storage orders?

I think you should check for the type of building first.

Warehouses, granaries, Markets, docks, some temples, Supply post, taverns have market orders. Others don't.

Reply:
I thought about it , that would be easy sure, but I dont thought this would be neccessary. As it is, it just will open the info window instead of storage orders (as described), so basically it would do the same as a right click on the given (building)tile (more versatile in my opinion). I thought this would provide more use...tell me, do you want me to limit it to just buidings which are set to "storage_show_special_orders"?

Reply:
They have special orders if their pantheon is active, otherwise they don't

Reply:
So is this working fine? Can I merge?